pkg/odo/genericclioptions/context.go

```diff
@@ -163,19 +163,35 @@ func getValidEnvInfo(command *cobra.Command) (*envinfo.EnvSpecificInfo, error) {
 	return envInfo, nil
 }
 
+func GetContextFlagValue(command *cobra.Command) string {
```

Comment:
Do we need this? Can we not use or modify `AddContextFlag()` instead?

Reply:
I thinking adding ContextFlag happens before parsing and this opperation should happen after the flags are parsed